### ISSUE TITLE
please remove code outline from dependent extensions

### ISSUE THREAD

**Issue by TheSpyder** [reasonml-editor/vscode-reasonml#140]
I'm not a huge fan of the code outline plugin; due to the way vscode opens new windows small and handles window resizing, the plugin winds up at 50-60% of my sidebar on new windows (which I do fairly regularly).

I wouldn't be so annoyed if I could disable the reason plugin except in my specific reason projects, but vscode doesn't support that.

Can you please change it to strongly recommended in the docs instead?

**Comment by freebroccolo** [comment]
Thanks for the suggestion. I will consider trying to make this optional. I have not looked at the configuration options recently but it may be possible to adjust the way that it behaves so that it doesn't use up so much of your sidebar by default. It also might be worth bringing this up on the repo for that extension.

**Comment by freebroccolo** [comment]
> I wouldn't be so annoyed if I could disable the reason plugin except in my specific reason projects, but vscode doesn't support that.

@TheSpyder I don't quite understand what you mean by this. You should be able to disable the plugin itself in the extension installation pane. Does that not work?

**Comment by TheSpyder** [comment]
@freebroccolo#5971 it's listed as a required plugin, which vscode enforces and I can't disable code outline without disabling reason first.

I just took a look at the config options and there's nothing about size adjustments, only the contents of the pane.

The problem wasn't that it took up much room by default; it's that when expanding the window vscode retained the explorer panel size (expanding code outline) rather than the other way around.

However it seems _something_ has changed since I logged this as it's now loading as a collapsed pane by default. I'll leave it enabled for a few days and see if that's permanent or just a cached setting with my workspaces.
